templates/OperationHeaderSnippet.stencil

```diff
@@ -5,7 +5,7 @@ headers["{{ header.key }}"] = {{ header.value }}
 {% endfor %}
 
 {% for header in op.optionalHeaders %}
-if {{ header.paramName }} != nil {
+if let {{ header.paramName }} = {{ op.clientMethodOptionsViewModel.variableName }}?.{{ header.paramName }} {
```

Comment:
Since `op` is already a view model, I think we can shorten the `clientMethodOptionsViewModel` name to something like `options`. :smile:

I don't think we've append "viewModel" to any other variable names in other view models. 
```Swift
    // Example from OperationViewModel
    let optionalQueryParams: [KeyValueViewModel]?
    let optionalHeaders: [KeyValueViewModel]?
    let pipelineContext: [KeyValueViewModel]?
```

So I think we can follow that precedent.

Reply:
I think this one was missed.